html/create-user.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Create User</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="../css/style.css" />
+</head>
+<body>
+    <nav>
+        <ul>
+            <li><a href="user-profile.html">User Profile</a></li>
+            <li><a href="user-list.html">User List</a></li>
+            <li><a href="create-user.html">Create User</a></li>
+            <li><a href="edit-user.html">Edit User</a></li>
+        </ul>
+    </nav>
+    <form>
+        <fieldset>
+            <div>
+                <label for="firstName">First name </label>
+                <input type="text" name="firstName" id="firstName">
+            </div>
+            <div>
+                <label for="lastName">Last name </label>
+                <input type="text" name="lastName" id="lastName">
+            </div>
+            <div>
+                <label for="phone">Phone </label>
+                <input type="text" name="phone" id="phone">
+            </div> 
+            <div>    
+                <label for="email">Email </label>
+                <input type="text" name="email" id="email">
+            </div>    
+            <div>
+                <input type="submit" placeholder="Submit">
```

Comment:
I can't give you specific reasons at the moment but if you have no compelling reason to use <input type="button">, I'd just use `<button type="submit">` here.

You could also put the button inside of the form instead of its own. It probably doesn't need it's own div.